### DISCUSSION TITLE
How can I migrate my data from one desktop to another?

### DISCUSSION THREAD

**Discussion by jie-zhang** [TriliumNext/Trilium#3403] [General]
I installed Trilium Notes as as desktop application with lots of pictures. Now I bought a new machine. I want to migrate my data to this new one, how could I do?

export/import md files can't do this, because the pictures doesn't involved in.

can I copy the db file and import it on another desktop?  or the only way is build a server and sync it ?

can any one help me with this question? thanks in advance.

**Comment by XXXJetfireXXX** [comment]
Yeah, you can just copy and paste `document.db` to the same path of a new machine, replacing an empty one.
Note that before launching Trilium after replacing document.db file you also need to delete .db-shm and .db-wal files.

**Reply by jie-zhang** [reply]
Thanks a lot for your patient and detailed answer!!!

**Reply by cw0rley** [reply]
Every time I replace the document.db and restart the app the startup screen comes up.
